conf/solr/conf/synonyms.txt

```diff
@@ -19,10 +19,120 @@ fooaaa,baraaa,bazaaa
 
 # Some synonym groups specific to this example
 GB,gib,gigabyte,gigabytes
-MB,mib,megabyte,megabytes
-Television, Televisions, TV, TVs
 #notice we use "gib" instead of "GiB" so any WordDelimiterGraphFilter coming
 #after us won't split it into two words.
+MB,mib,megabyte,megabytes
+Television, Televisions, TV, TVs
+Volume, Vol., Vol
+&, and
+1, One, I
```

Comment:
We should test this one; this might cause false positives. Adding `Vol. 1, Vol. I` might be a more robust approach? But needs some testing. If Conail has some time, he should be able to load this up to run it!